bodhi/server/consumers/masher.py

```diff
@@ -1284,12 +1289,43 @@ def _raise_on_get_build_multicall_error(self, result, build):
             self.log.error(err)
             raise Exception(err)
 
+    def _add_build_to_newest_builds(self, newest_builds, koji_build, override=False):
+        """
+        Add Koji build to newest_builds dict if it's newer than the one there or override is set.
+
+        Args:
+            newest_builds (dict): Dict with name:stream as a key and version as a value.
```

Comment:
It looks like the value is actually a dictionary with the key "version" (perhaps among others?) based on reading the code below:

```
curr_version = newest_builds[ns]['version']
```